drivers/periph_common/eeprom.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers
+ * @{
+ *
+ * @file
+ * @brief       Common eeprom functions implementation
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <string.h>
+#include "cpu.h"
+#include "assert.h"
+
+/* guard this file, must be done before including periph/eeprom.h */
+#if defined(EEPROM_SIZE)
+
+#include "periph/eeprom.h"
+
+size_t eeprom_read(uint32_t pos, uint8_t *data, size_t len)
+{
+    assert(pos + len < EEPROM_SIZE);
+
+    for (size_t i = 0; i < len; ++i) {
```

Comment:
here and elsewhere: please use ```i++```.

Reply:
Done and hope this is the last one ;)

Reply:
@kaspar030 why, if it makes a difference I always thought `++i` is better than `i++`

Reply:
1. consistency. In RIOT, 'for.*i++' vs 'for.*++i' is about four to one.
2. In C, it doesn't make a difference. BUT "for(...;i++)' is considered ideomatic C, and thus more readable.

Reply:
Ok, I'm fine with that. Now it should be all good here, right ?